src/pyuvdata/telescopes.py

```diff
@@ -95,6 +95,7 @@
             height=1019.222 * units.m,
         ),
         "antenna_diameters": 6.1,
+        "citation": "Private communication (2024)",
```

Comment:
can we get at least a name here so we know who provided the info?

Reply:
Done